CoreUtilities/test/unit/src/au/gov/asd/tac/constellation/utilities/icon/DefaultCustomIconProviderNGTest.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.utilities.icon;
+
+import au.gov.asd.tac.constellation.utilities.color.ConstellationColor;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.List;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.times;
+import org.openide.util.Exceptions;
+import org.openide.util.ImageUtilities;
+import static org.testng.Assert.assertEquals;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ * Test class for DefaultCustomIconProvider
+ *
+ * @author Delphinus8821
+ */
+public class DefaultCustomIconProviderNGTest {
+
+    final static String TEST_ICON_NAME = "TestIcon2";
+
+    public DefaultCustomIconProviderNGTest() {
+    }
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+    }
+
+    @BeforeMethod
+    public void setUpMethod() throws Exception {
+    }
+
+    @AfterMethod
+    public void tearDownMethod() throws Exception {
+    }
+
+    /**
+     * Runner for tests
+     */
+    @Test(expectedExceptions = NullPointerException.class)
+    public void runTests() {
+        try ( MockedStatic<DefaultCustomIconProvider> defaultCustomIconProviderMock = Mockito.mockStatic(DefaultCustomIconProvider.class)) {
+            try ( MockedStatic<IconManager> iconManagerMock = Mockito.mockStatic(IconManager.class)) {
```

Comment:
you're actually able to combine try-with-resources (just separate each resource with a semi-colon, i.e. `try (MockedStatic<type1> foo = ...; MockedStatic<type2> bar = ...;)`